content/nim/monitoring/security-monitoring/configure/set-up-app-protect-instances.md

```diff
@@ -32,7 +30,7 @@ Complete the following prerequisites before proceeding with the steps in this gu
    {{< include "nim/tech-specs/security-data-plane-dependencies.md" >}}
 
 1. Determine your use case: **Security Monitoring only** or **Security Monitoring and Configuration Management**.
-1. [Install the NGINX Security Monitoring module]({{< relref "/nim/monitoring/security-monitoring/install-security-monitoring.md" >}}) and [upload your license]({{< relref "/nim/admin-guide/license/add-license.md" >}}).
+1. [Install the Security Monitoring module]({{< relref "/nim/monitoring/security-monitoring/install-security-monitoring.md" >}}) and [upload your license]({{< relref "/nim/admin-guide/license/add-license.md" >}}).
```

Comment:
Since Sec Mon is installed w/ NIM 2.19, this guide will be removed.

That update is in the `docs-1604-decouple-security-monitoring` branch, out for review. You might want to rebase to that branch. I also removed the Sec Mon RNs.

Here's the related PR: https://github.com/nginx/documentation/pull/86